### PR TITLE
PR | Squashed commit for git training

### PR DESCRIPTION
**Pull request by IsmagilovQA**
Please review my PR. Leave comments if you see something wrong. I will appreciate it!

### CODE REVIEW

**Review by andreyzavrichko — CHANGES_REQUESTED** [review]
new comment

File: src/test/java/domain/com/homework_9/TestClassForGit.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package domain.com.homework_9;
+
+import org.junit.jupiter.api.Test;
+
+import java.sql.SQLOutput;
+
+public class TestClassForGit {
+
+    /**
+     * Here can be some comments.
```
Comment (andreyzavrichko) [inline src/test/java/domain/com/homework_9/TestClassForGit.java:10]: new comment

**Comment by IsmagilovQA** [comment]
> new comment

Fixed. Please check once again please.

**Review by andreyzavrichko — APPROVED** [review]
nice work!

**Pull request merged**